lob/version.py

```diff
@@ -1,3 +1,3 @@
 from __future__ import unicode_literals
 
-VERSION = '3.4.0'
+VERSION = '4.0.0'
```

Comment:
It doesn't look like we're changing any behavior to the library, is there a specific reason why we need to bump the version?

Reply:
`4.0.0` was released back in October, but the code wasn't updated. Whomp whomp. This PR just brings us into sync with reality.